cmd/config-seed/main.go

```diff
@@ -71,7 +74,7 @@ func bootstrap(profile string) {
 				if ok {
 					config.LoggingClient.Error(e.Error())
 				} else {
-					return
```

Comment:
This is a return from the Go func, not the service. I don'r think we want to exit the service here.